official-git/run-tests.sh

```diff
@@ -52,47 +54,91 @@ GIT_SKIP_TESTS="$GIT_SKIP_TESTS t1308.2[6-7]" # No support for line ending handl
 GIT_SKIP_TESTS="$GIT_SKIP_TESTS t2018.[6-7] t2018.[9] t2018.1[5-8]"
 GIT_SKIP_TESTS="$GIT_SKIP_TESTS t3000.7"
+GIT_SKIP_TESTS="$GIT_SKIP_TESTS t5510.[4-9] t5510.[1-7][0-9]" # Just the basic fetch tests are working for now
 export GIT_SKIP_TESTS
 
-make t0000-basic.sh \
-        t0004-unwritable.sh \
-        t0007-git-var.sh \
-        t0008-ignores.sh \
-        t0010-racy-git.sh \
-        t0062-revision-walking.sh \
-        t0070-fundamental.sh \
-        t0081-line-buffer.sh \
-        t1000-read-tree-m-3way.sh \
-        t1001-read-tree-m-2way.sh \
-        t1002-read-tree-m-u-2way.sh \
-        t1003-read-tree-prefix.sh \
-        t1004-read-tree-m-u-wf.sh \
-        t1005-read-tree-reset.sh \
-        t1008-read-tree-overlay.sh \
-        t1009-read-tree-new-index.sh \
-        t1011-read-tree-sparse-checkout.sh \
-        t1012-read-tree-df.sh \
-        t1014-read-tree-confusing.sh \
-        t1100-commit-tree-options.sh \
-        t1304-default-acl.sh \
-        t1308-config-set.sh \
-        t1403-show-ref.sh \
-        t1500-rev-parse.sh \
-        t2000-checkout-cache-clash.sh \
-        t2001-checkout-cache-clash.sh \
-        t2002-checkout-cache-u.sh \
-        t2006-checkout-index-basic.sh \
-        t2009-checkout-statinfo.sh \
-        t2014-switch.sh \
-        t2018-checkout-branch.sh \
-        t2100-update-cache-badpath.sh \
-        t3000-ls-files-others.sh \
-        t3002-ls-files-dashpath.sh \
-        t3006-ls-files-long.sh \
-        t3020-ls-files-error-unmatch.sh \
-        t3800-mktag.sh \
-        t4113-apply-ending.sh \
-        t4123-apply-shrink.sh \
-        t7062-wtstatus-ignorecase.sh \
-        t7511-status-index.sh
+echo t0000-basic
+./t0000-basic.sh
+echo t0004-unwritable
+./t0004-unwritable.sh
+echo t0007-git-var
+./t0007-git-var.sh
+echo t0008-ignores
+./t0008-ignores.sh
+echo t0010-racy-git
+./t0010-racy-git.sh
+echo t0062-revision-walking
+./t0062-revision-walking.sh
+echo t0070-fundamental
+./t0070-fundamental.sh
+echo t0081-line-buffer
+./t0081-line-buffer.sh
+echo t1000-read-tree-m-3way
+./t1000-read-tree-m-3way.sh
+echo t1001-read-tree-m-2way
+./t1001-read-tree-m-2way.sh
+echo t1002-read-tree-m-u-2way
+./t1002-read-tree-m-u-2way.sh
+echo t1003-read-tree-prefix
+./t1003-read-tree-prefix.sh
+echo t1004-read-tree-m-u-wf
+./t1004-read-tree-m-u-wf.sh
+echo t1005-read-tree-reset
+./t1005-read-tree-reset.sh
+echo t1008-read-tree-overlay
+./t1008-read-tree-overlay.sh
+echo t1009-read-tree-new-index
+./t1009-read-tree-new-index.sh
+echo t1011-read-tree-sparse-checkout
+./t1011-read-tree-sparse-checkout.sh
+echo t1012-read-tree-df
+./t1012-read-tree-df.sh
+echo t1014-read-tree-confusing
+./t1014-read-tree-confusing.sh
+echo t1100-commit-tree-options
+./t1100-commit-tree-options.sh
+echo t1304-default-acl
+./t1304-default-acl.sh
+echo t1308-config-set
+./t1308-config-set.sh
+echo t1403-show-ref
+./t1403-show-ref.sh
+echo t1500-rev-parse
+./t1500-rev-parse.sh
```

Comment:
didn't I recently add t1503-rev-parse-verify too?

Reply:
I'm not seeing it in master branch here. Maybe it's in the other PR?

Reply:
Shit.. it looks like #230 did everything *except* add it to Travis. Can you just make sure it's still passing and add it here since you're refactoring it anyways?

Reply:
Sure, not a problem.

Reply:
It's failing on one of the 10 tests. Should I ignore that test or leave it as another PR?

Reply:
There should already be an ignore for it that was added in #230, it just wasn't added to the list of tests. (In fact, I think there should be 3 tests failing..)

Reply:
The 't' prefix was missing for the number 10 test. I'll have a commit pushed shortly. I'm just making sure that the builds always pass and there's no more intermittent failures.

Reply:
Okay, let me know when you're confident and I'll merge.